scripts/run_app_paas.sh

```diff
@@ -102,11 +67,7 @@ check_params
 
 trap "on_exit" EXIT
 
-configure_aws_logs
-
 # The application has to start first!
 start_application "$@"
```

Comment:
Part of me thinks we can/should kill this whole script now. I don't think the disk check is as useful now that we're not writing things to disk constantly.

Reply:
Yeah, removing it sounds reasonable to me.